packages/story-editor/src/app/media/utils/useMediaUploadQueue/index.js

```diff
@@ -321,6 +321,13 @@ function useMediaUploadQueue() {
 
           const trackTiming = getTimeTracker('load_upload_media');
 
+          if (resource?.baseColor) {
+            additionalData.meta = {
+              ...additionalData.meta,
+              web_stories_base_color: resource.baseColor,
+            };
```

Comment:
```suggestion
            additionalData.meta.web_stories_base_color = resource.baseColor;
```

Reply:
No, can't do this, as meta is always set / an object.

Reply:
Could theoretically do

```suggestion
						additionalData.meta = additionalData.meta || {};
            additionalData.meta.web_stories_base_color = resource.baseColor;
```

but both work

Reply:
Honestly don't mind either way, spread operator made sense to me and left very readable, but willing to change it.